drivers/SmartThings/zigbee-motion-sensor/src/aqara/aqara_utils.lua

```diff
@@ -0,0 +1,186 @@
+local capabilities = require "st.capabilities"
+local cluster_base = require "st.zigbee.cluster_base"
+local data_types = require "st.zigbee.data_types"
+
+local aqara_utils = {}
+
+local detectionFrequency = capabilities["stse.detectionFrequency"]
+aqara_utils.detectionFrequency = detectionFrequency
+local detectionFrequencyId = "stse.detectionFrequency"
+aqara_utils.detectionFrequencyId = detectionFrequencyId
+local detectionFrequencyCommand = "setDetectionFrequency"
+aqara_utils.detectionFrequencyCommand = detectionFrequencyCommand
+local sensitivityAdjustment = capabilities["stse.sensitivityAdjustment"]
+aqara_utils.sensitivityAdjustment = sensitivityAdjustment
+local sensitivityAdjustmentId = "stse.sensitivityAdjustment"
+aqara_utils.sensitivityAdjustmentId = sensitivityAdjustmentId
+local sensitivityAdjustmentCommand = "setSensitivityAdjustment"
+aqara_utils.sensitivityAdjustmentCommand = sensitivityAdjustmentCommand
+
+local PRIVATE_CLUSTER_ID = 0xFCC0
+aqara_utils.PRIVATE_CLUSTER_ID = PRIVATE_CLUSTER_ID
+local PRIVATE_ATTRIBUTE_ID = 0x0009
+aqara_utils.PRIVATE_ATTRIBUTE_ID = PRIVATE_ATTRIBUTE_ID
+local MFG_CODE = 0x115F
+aqara_utils.MFG_CODE = MFG_CODE
+
+local MOTION_ILLUMINANCE_ATTRIBUTE_ID = 0x0112
+aqara_utils.MOTION_ILLUMINANCE_ATTRIBUTE_ID = MOTION_ILLUMINANCE_ATTRIBUTE_ID
+local FREQUENCY_ATTRIBUTE_ID = 0x0102
+aqara_utils.FREQUENCY_ATTRIBUTE_ID = FREQUENCY_ATTRIBUTE_ID
+local SENSITIVITY_ATTRIBUTE_ID = 0x010C
+aqara_utils.SENSITIVITY_ATTRIBUTE_ID = SENSITIVITY_ATTRIBUTE_ID
+
+local MOTION_DETECTED_UINT32 = 65536
+aqara_utils.MOTION_DETECTED_UINT32 = MOTION_DETECTED_UINT32
+local MOTION_DETECTED_NUMBER = 1
+aqara_utils.MOTION_DETECTED_NUMBER = MOTION_DETECTED_NUMBER
+
+local UNOCCUPIED_TIMER = "unoccupiedTimer"
+aqara_utils.UNOCCUPIED_TIMER = UNOCCUPIED_TIMER
+
+local PREF_CHANGED_KEY = "prefChangedKey"
```

Comment:
anything you're only using in this utils file doesn't need to be put into the map that gets loaded by the two sub-drivers

Reply:
Delete duplicate codes.